src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
Maybe, but for example the feature you're adding now (requesting a notification resend) is not documented anywhere